bibtexparser/tests/test_bparser.py

```diff
@@ -12,6 +12,13 @@
 from bibtexparser import customization
 
 
+NONSTANDARD_ENTRY = """@nonstandard{mycitekey,
```

Comment:
Why was this variable introduced? It doesn't seem to be used in this file.

Reply:
My bad! Thanks @mwatts15.